pkg/config/config.go

```diff
@@ -1024,6 +1024,10 @@ func InitConfig(config Config) {
 	// Serverless Agent
 	config.BindEnvAndSetDefault("serverless.logs_enabled", true)
 	config.BindEnvAndSetDefault("enhanced_metrics", true)
+	config.BindEnvAndSetDefault("trace_enabled", true)
+	config.BindEnvAndSetDefault("capture_lambda_payload", false)
+	config.BindEnv("dotnet_tracer_home")
+	config.BindEnv("service")
```

Comment:
I am very surprised that DD_SERVICE is not already registered since it's a very common env variable used outside of the serverless environment